src/Collection/CollectionTrait.php

```diff
@@ -885,9 +892,10 @@ public function _unwrap()
     }
 
     /**
-     * {@inheritDoc}
```

Comment:
The `{@inheritDoc}` needs to be kept for inheriting the description.

Reply:
there is nothing to inherit. i didnt find any code.